sdk/monitor/Azure.Monitor.Ingestion/src/Models/UploadLogsOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Serialization;
+
+namespace Azure.Monitor.Ingestion
+{
+    /// <summary>
+    /// The options model to configure the request to upload logs to Azure Monitor.
+    /// </summary>
+    public class UploadLogsOptions
+    {
+        /// <summary>
+        /// The serializer to use to convert the log objects to JSON.
+        /// </summary>
+        public ObjectSerializer ObjectSerializer { get; set; }
+
+        /// <summary>
+        /// The max concurrent requests to send to the Azure Monitor service when uploading logs.
+        /// </summary>
+        public int MaxConcurrency { get; set; }
```

Comment:
Do we need to have any validation here to ensure users don't enter invalid values, e.g. negative numbers.

Reply:
will add <= 0 checking to method